### PR TITLE
CMIML-149-fature/tabs

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

File: src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import Tab from '@mui/material/Tab';
+
+import { TabPanel } from './TabPanel';
+import { StyledTabs } from './Tabs.style';
+import { TabsList } from './TabsList';
+
+export const Tabs = () => {
+  const [tabIndex, setTabIndex] = useState(0);
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
```
Comment (IhorNesterenko99) [inline src/components/Tabs/Tabs.tsx:11]: Could you please import SynteticEvent from React on top?

File: src/components/Tabs/TabPanel/TabPanel.style.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { styled } from '@mui/system';
+import { Box } from '@mui/material';
+
+import { variables } from 'styles/variables';
+import theme from 'styles/theme';
+
+export const StyledPanel = styled(Box)`
+  background: ${variables.palette.grayBG};
+  padding: ${theme.spacing(2.4, 2.4, 3.5)};
+  border-top: 1px solid ${variables.palette.gray70};
```
Comment (IhorNesterenko99) [inline src/components/Tabs/TabPanel/TabPanel.style.tsx:10]: Could you pls use borderWidth variable from variables?

File: src/components/Tabs/TabPanel/TabPanel.types.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export type TabPanelProps = {
+  children?: React.ReactNode;
```
Comment (IhorNesterenko99) [inline src/components/Tabs/TabPanel/TabPanel.types.ts:2]: Could you pls import ReactNode type from React on the top?

File: src/components/Tabs/TabsList.tsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { Icon } from 'components/Icon';
+
+export const TabsList = [
+  { label: 'Applets', icon: <Icon.Applets />, content: 'content 1' },
```
Comment (IhorNesterenko99) [inline src/components/Tabs/TabsList.tsx:4]: It seems we need French interpretation for all of the strings.

File: src/styles/variables/palette.ts [hunk 5]
```diff
@@ -18,7 +18,10 @@ export const palette = {
   shades100_alfa50: 'rgba(0, 0, 0, 0.5)',
   shades100_alfa87: 'rgba(0, 0, 0, 0.87)',
 
+  grayBG: '#FDFCFF',
+  gray70: '#DEE3EB',
   gray80: '#526070',
+  gray95: '#1A1C1E',
```
[the remark this answers is not in the document]
Comment (kdanilenka32) [reply src/styles/variables/palette.ts:22]: changed according the design

File: src/components/Tabs/TabPanel/TabPanel.types.ts (hunk 3 above)
Comment (kdanilenka32) [reply src/components/Tabs/TabPanel/TabPanel.types.ts:2]: not necessary here

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**